### PR TITLE
[fluentd-elasticsearch] (PREVIEW - DO NOT MERGE) Add suppressTypeName parameter to eliminate ES 7 deprecation warnings

### LINKED ISSUE #381: [fluentd-elasticsearch] Fluentd prints continuous deprecation warning against ES 7.3: "[types removal] Specifying types in bulk requests is deprecated."
**Is this a request for help?**:
No.

---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):
BUG REPORT

**Version of Helm and Kubernetes**:
```
$ helm version
version.BuildInfo{Version:"v3.0.3", GitCommit:"ac925eb7279f4a6955df663a0128044a8a6b7593", GitTreeState:"clean", GoVersion:"go1.13.7"}
```

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T23:34:25Z", GoVersion:"go1.14.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16+", GitVersion:"v1.16.8-eks-e16311", GitCommit:"e163110a04dcb2f39c3325af96d019b4925419eb", GitTreeState:"clean", BuildDate:"2020-03-27T22:37:12Z", GoVersion:"go1.13.8", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart in which version**:
3.0.2

**What happened**:
When this helm chart is configured to point at Elasticsearch 7.3.2, the fluentd pods prints out the following message approximately once per second:
```
warning: 299 Elasticsearch-7.3.2-1c1faf1 "[types removal] Specifying types in bulk requests is deprecated."
```

**What you expected to happen**:
The pod should not continuously print out this warning.

**How to reproduce it** (as minimally and precisely as possible):
Deploy this helm chart pointing at an ES 7.3.2 domain.

**Anything else we need to know**:
The Elasticsearch plugin this chart uses is aware of this issue and has a flag which is supposed to stop the warnings: https://github.com/uken/fluent-plugin-elasticsearch#suppress_type_name

This flag should probably be added to this chart as a recommended option for people running against ES 7.x.

### PR DESCRIPTION
**Pull request by Ghazgkull**
#### What this PR does / why we need it:

This PR adds `elasticsearch.suppressTypeName` as a chart parameter in order to leverage the `suppress_type_name` option on the fluentd-elasticsearch plugin.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #381

#### Special notes for your reviewer:

Please note that this PR will not work at this moment! At the time this PR is being opened, the referenced fluend-elasticsearch image version (3.0.4) is in PR but not yet been published. (https://github.com/kubernetes/kubernetes/pull/93116)

I'm submitting this PR now so it can be reviewed and I can incorporate any feedback as needed.

Note that I have tested both the new fluentd-elasticsearch image 3.0.4 (published as a fork) as well as the changes I'm submitting here (published to a private helm repo), so I'm confident that everything will work once the k8s PR has been merged.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[fluentd-elasticsearch]`)

### CODE REVIEW

File: charts/fluentd-elasticsearch/templates/configmaps.yaml [hunk 1]
```diff
@@ -531,6 +531,7 @@ data:
       reload_on_failure "#{ENV['OUTPUT_RELOAD_ON_FAILURE']}"
       reload_connections "#{ENV['OUTPUT_RELOAD_CONNECTIONS']}"
       request_timeout "#{ENV['OUTPUT_REQUEST_TIMEOUT']}"
+      suppress_type_name "#{ENV['OUTPUT_SUPPRESS_TYPE_NAME']}"
```
Comment (monotek) [inline charts/fluentd-elasticsearch/templates/configmaps.yaml:534]: I'd like to see a "{{- if .Values.elasticsearch.suppressTypeName }}" here, to get sure not to change stuff for people who are updateing.

File: charts/fluentd-elasticsearch/templates/daemonset.yaml [hunk 2]
```diff
@@ -151,6 +151,8 @@ spec:
           value: {{ .Values.elasticsearch.reloadConnections | quote }}
         - name: OUTPUT_REQUEST_TIMEOUT
           value: {{ .Values.elasticsearch.requestTimeout | quote }}
+        - name: OUTPUT_SUPPRESS_TYPE_NAME
```
Comment (monotek) [inline charts/fluentd-elasticsearch/templates/daemonset.yaml:154]: I'd like to see a "{{- if .Values.elasticsearch.suppressTypeName }}" here, to get sure not to change stuff for people who are updateing.

File: charts/fluentd-elasticsearch/Chart.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: fluentd-elasticsearch
-version: 9.5.1
+version: 9.5.2
```
Comment (monotek) [inline charts/fluentd-elasticsearch/Chart.yaml:3]: New feature should change version to 9.6.0

**Comment by Ghazgkull** [comment]
PR comments addressed. New commit pushed.

**Comment by monotek** [comment]
I've just pushed image 3.0.4.
https://quay.io/repository/fluentd_elasticsearch/fluentd?tab=tags

**Pull request merged**